src/UI/pages/dataHub.tsx

```diff
@@ -16,7 +16,13 @@ function DataHub() {
         >
           <SectionPanel title="Data Hub Page">
             <AuthWrapper role="">
-              <DataHubPanel />
+              <div>
+                <main>
+                  <Container>
```

Comment:
We don't need these here either - they're in lines 9-11 already

Reply:
Sorry Andrew, it's ok now. Please, can you review ?